packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -134,6 +135,16 @@ class ScaffoldPrelayoutGeometry {
   /// will be 0.0.
   final EdgeInsets minInsets;
 
+  /// The minimum padding to inset interactive elements to be within a safe,
+  /// un-obscured space.
+  ///
+  /// This value is the result of calling [MediaQuery.viewPadding] in the
```

Comment:
Is the value just based on [MediaQuery.viewPadding] or is it the same as viewPadding?

Reply:
It's also dependent on `_resizeToAvoidBottomInset`, I'll update the docs. :)